client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/maps-panel/upload-map-dialog/upload-map-dialog.component.html

```diff
@@ -9,7 +9,7 @@ <h1 mat-dialog-title>Upload Map</h1>
     </mat-form-field>
   </div>
   <div class="dialog-actions" mat-dialog-actions>
-  	<button mat-raised-button color="primary" [mat-dialog-close]="returnResult()" (disabled)="file === null">Save</button>
+  	<button mat-raised-button color="primary" [mat-dialog-close]="returnResult()" [disabled]="file === null || title === null || title === ''">Save</button>
```

Comment:
probably we can simplify this with "[disabled]="file === null || !title" ?

Reply:
Yep that'd be fine for this case, I don't think we're going to change it in the future to maintain empty but detect nulls/undefined